composer.json

```diff
@@ -15,8 +15,8 @@
         "psr-4": { "M6Web\\Component\\": "src/" }
     },
     "require": {
-        "php"                         : ">=5.4.0",
-        "elasticsearch/elasticsearch" : "^2.0"
+        "php"                         : ">=5.6.6",
```

Comment:
Since we removed travis tests on 5.6 (because they failed), I think we can go forward and require `^7.0`